monitorix

```diff
@@ -24,7 +24,7 @@ require 5.006;
 use strict;
 use warnings;
 use FindBin qw($Bin);
-use lib $Bin . "/lib", "/usr/lib/monitorix";
+use lib $Bin . "/lib", "/usr/local/share/monitorix";
```

Comment:
Why you need to do this change?
Monitorix modules are currently installed on `/usr/lib/monitorix` directory. You cannot change this without forcing to everybody to change the installation directory.

Reply:
oh, yes, (shame on me) it is changed during installation on FreeBSD since all nonsystem stuff is installed into `/usr/local/`
I'll change it of course to the original `/usr/lib`